test/external-modules/die-with-dignity/build.gradle

```diff
@@ -14,17 +14,7 @@ GradleUtils.extendSourceSet(project, "main", "javaRestTest", tasks.named("javaRe
 
 tasks.named("javaRestTest").configure {
   it.onlyIf("snapshot build") { BuildParams.isSnapshotBuild() }
-  systemProperty 'tests.security.manager', 'false'
-  systemProperty 'tests.system_call_filter', 'false'
-  nonInputProperties.systemProperty 'log', testClusters.named("javaRestTest").map(c -> c.singleNode().getServerLog())
-  systemProperty 'runtime.java.home', BuildParams.runtimeJavaHome
-}
-
-testClusters.matching { it.name == "javaRestTest" }.configureEach {
-  systemProperty "die.with.dignity.test", "true"
-  // disable exit on out of memory error to let DieWithDignityIT verify that OOM handling without that works (including OOMs that are not caused by
-  // memory like native threads. We leave it to the JVM to test that exit on OOM works via the flag.
-  jvmArgs '-XX:-ExitOnOutOfMemoryError'
+  usesDefaultDistribution()
```

Comment:
Is it strictly necessary for these tests to use the default distribution? Could they not use the integ-test distro?